### PR TITLE
Working with the superhero API

### LINKED ISSUE #3989: [API]Working with superhero API
#### Proposed title of article
[API]Working with superhero API

#### Proposed article introduction
The superhero API is a collection of data from all the superheroes from the comic world. It consists of information such as name,  Intelligence,  Strength,   Speed,   Durability,   Power,   Combat among others. This information is presented in JSON format for accessibility.

This article focuses on building an application that fetches information about all the superheroes and display them efficiently. We will build a web application to consume the API data with bootstrap styling. Should this article be published,  the reader will be able to consume an API endpoints in their projects.
 
#### Key takeaways 
1. Working with JSON
2. Data presentation from an API
3. Consuming the Superhero API data

#### Article quality
This article covers an API that most developers have not tried working with. Besides, most APIs are only tried using Postman and Insomnia but the real implementation in a web app is rare. For this reason, this article illustrates how to consume Superhero API in webpage and provided addition bootstrap styling. It emulates working on a full-stack web project with a backed but in this case we have data already.

#### References
- https://superheroapi.com/

### PR DESCRIPTION
**Pull request by victorelvice**
> If you have not already, please go over our [Resources Page](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md) for more tips and tricks on how to improve your overall technical writing so reviews are swifter, increase the chance of a payout and provide more value to our readers.

> For first-time contributors and for more details on our submission guidelines, **see our [Contributing Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**.

#### Formatting and structure checks
- [x] Have you used the correct folder and file structure? - **see [Contribution Guidelines](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/CONTRIBUTING.md)**
- [x] Is your article properly formatted in Markdown? - **see [Github Markdown guide](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet)**
- [x] Have you used our preferred heading structure? - **[H3 (###) and up](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#headers)**
- [x] Does your article follow any of the suggested structure formats? - **see [example formats](https://github.com/section-io/engineering-education/blob/master/CONTRIBUTING.md#example-format-structure)**

- [x] Is your article over 750 words? - **Tool**: [Wordcounter.net](https://wordcounter.net)
- [x] Is the article you are submitting an in-depth and unique article? Does it go beyond what is in the official docs and what is covered in other blog sites. See these articles as [examples](https://github.com/section-engineering-education/engineering-education/blob/master/new_contributors/resources-page.md).
- [x] Does your article provide enough value and detail about your topic? - **Articles should be clear, accurate, and fully explained.**
- [x] Can your article be understood by beginners? - **Assume the audience is smart but has no prior exposure to the common terminology in your article.**

- [x] Have you included a hero image that is **600x400 pixels**, under **300KB in size**, and have the copyright to use it? - **Tools:** [Pixlr Image Editor](https://pixlr.com/e) for resizing and [Unsplash](https://unsplash.com) for Creative Commons images
      
#### Grammar & plagiarism checks
- [x] Have you spellchecked and grammar checked your article? - **Tools:** [Write&Improve](https://writeandimprove.com/?lang=en-GB) - [ProWritingAid](https://prowritingaid.com/) - [Grammarly](https://grammarly.com) 
- [x] Please place your article through a 3rd party plagiarism checker. We suggest using Quetext, [this tool](https://www.quetext.com/) is free and has a daily limit. **We typically accept articles with 10% or less.**
- [x] Have you checked your article for readability? - **Tool:** [Hemmingway](http://www.hemingwayapp.com)
- [x] Have you added sources for quotes and images that aren't yours?

#### Technical checks
- [x] Are your code snippets properly formatted for syntax highlighting - **see [Syntax guide](https://gohugo.io/content-management/syntax-highlighting)**
- [x] Have you checked your code runs correctly and you've highlighted all necessary dependencies for installation?
- [x] Are the software programs and packages you're highlighting in your article up to date, using current versions and not deprecated?

This closes #3989

> Finally, delete the article checklist notes in blockquotes and submit your PR. We look forward to reviewing your article.

### CODE REVIEW

**Comment by Kayere** [comment]
Plagiarism check done ✅

**Review by mercymeave — COMMENTED** [review]
Great piece here @victorelvice. :tada:  The community is campaigning for such straight to the point and easy to read articles :rocket: . Your article was clear and elaborate. Proceeding to  tech review.

File: content/articles/superhero-api/index.md [hunk 1]
```diff
@@ -0,0 +1,471 @@
...
+STATIC_URL = '/static/'
+
+STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
+
+STATICFILES_DIRS = (
+    os.path.join(BASE_DIR, 'static'),
+)
+```
+
+You can  find the css and js file in [this link]
```
Comment (mercymeave) [inline content/articles/superhero-api/index.md:463]: @victorelvice 

Provide this link. I think you must have missed it.

Thanks!

**Review by mercymeave — CHANGES_REQUESTED** [review]

**Comment by victorelvice** [comment]
@mercymeave 

Sorry I must have missed that.

**Review by mercymeave — APPROVED** [review]
@victorelvice 

Great work! :tada:

You have a very direct and easy to read the article. We encourage such articles from our authors. :rocket:

Moving forward to the final review.

Thanks and keep up :handshake:

**Comment by WanjaMIKE** [comment]
Nice work gents @victorelvice @mercymeave

**Pull request merged**